lib/malloc.cpp

```diff
@@ -229,13 +229,11 @@ namespace quda
 
```

Comment:
The memory handling for different targets might be different enough that it would be better to revert the changes to this file, and simply have malloc_cuda.cpp, and and a new variant allocate file when different targets are added.